src/research/lightning.json5

```diff
@@ -108,8 +108,31 @@
     },
     {
       name: 'Checkbox Toggle',
+      openUIName: 'Switch',
       url: 'https://www.lightningdesignsystem.com/components/checkbox-toggle',
       category: 'Component Blueprints',
+      concepts: [
+        {
+          name: 'basic',
+          description: 'Basic toggle',
+          image: 'lighting-toggle-basic.png',
+        },
+        {
+          name: 'checked',
+          description: 'Checked toggle',
+          image: 'lighting-toggle-checked.png',
+        },
+        {
+          name: 'disabled',
+          description: 'Disabled toggle',
+          image: 'lighting-toggle-disabled.png',
+        },
+        {
+          name: 'label',
+          description: 'Label associated with the toggle',
```

Comment:
Heads up, leave description out if the guide didn't have one.  This phase of work is solely about collecting the data.